lib/api/utilities/page_size_helper.rb

```diff
@@ -65,10 +59,7 @@ def resulting_page_size(value, relation = nil)
       # the largest option of per_page size,
       # or the magic fallback value 500.
       def maximum_page_size
```

Comment:
The problem with comments is that they become outdated :)

We could still leave that switch in here to avoid the per_page_options problem.